lib/solr_wrapper/instance.rb

```diff
@@ -222,11 +218,10 @@ def with_collection(options = {})
     end
 
     ##
-    # Clean up any files solr_wrapper may have downloaded
+    # Clean up any files solr_wrapper may have downloaded, except the solr-{ver}.zip file
     def clean!
       stop
       remove_instance_dir!
-      FileUtils.remove_entry(config.download_dir, true) if File.exist?(config.download_dir)
```

Comment:
Instead of removing this line whole-sale, let's put it behind a named parameter flag (say, `#clean! except_download_dir: true`, or something better). There's a real use-case for "get rid of everything solr_wrapper has done", and I want to preserve that ability.